<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockProfileDialog</class>
 <widget class="QDialog" name="ClockProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GWEpp: New clock profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_new_profile_name">
     <property name="text">
      <string>New profile name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_new_profile_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_header">
     <property name="text">
      <string>GPU clock offsets</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_gpu_offsets" rowstretch="0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="6">
      <widget class="QCheckBox" name="checkBox_pstate4">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBox_pstate1">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QSpinBox" name="spinBox_pstate6_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QSlider" name="verticalSlider_pstate7_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QSpinBox" name="spinBox_pstate5_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QCheckBox" name="checkBox_pstate6">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBox_pstate0">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinBox_pstate1_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_footer_2">
       <property name="text">
        <string>Clocks (MHz)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QSlider" name="verticalSlider_pstate4_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinBox_pstate0_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QCheckBox" name="checkBox_pstate5">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QSlider" name="verticalSlider_pstate3_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QCheckBox" name="checkBox_pstate7">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QSlider" name="verticalSlider_pstate6_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="checkBox_pstate2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QSpinBox" name="spinBox_pstate3_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_footer_3">
       <property name="text">
        <string>Offset (MHz)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLineEdit" name="lineEdit_pstate4_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_pstate1_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QSpinBox" name="spinBox_pstate7_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QCheckBox" name="checkBox_pstate3">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSlider" name="verticalSlider_pstate2_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QLineEdit" name="lineEdit_pstate7_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSlider" name="verticalSlider_pstate1_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_pstate0_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QLineEdit" name="lineEdit_pstate6_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="verticalSlider_pstate0_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="spinBox_pstate2_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QSlider" name="verticalSlider_pstate5_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="lineEdit_pstate3_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QLineEdit" name="lineEdit_pstate5_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QSpinBox" name="spinBox_pstate4_gpu_clock_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit_pstate2_gpu_max_clock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_new_profile_name</tabstop>
  <tabstop>checkBox_pstate0</tabstop>
  <tabstop>checkBox_pstate1</tabstop>
  <tabstop>checkBox_pstate2</tabstop>
  <tabstop>checkBox_pstate3</tabstop>
  <tabstop>checkBox_pstate4</tabstop>
  <tabstop>checkBox_pstate5</tabstop>
  <tabstop>checkBox_pstate6</tabstop>
  <tabstop>checkBox_pstate7</tabstop>
  <tabstop>verticalSlider_pstate0_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate1_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate2_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate3_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate4_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate5_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate6_gpu_clock_offset</tabstop>
  <tabstop>verticalSlider_pstate7_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate0_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate1_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate2_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate3_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate4_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate5_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate6_gpu_clock_offset</tabstop>
  <tabstop>spinBox_pstate7_gpu_clock_offset</tabstop>
  <tabstop>checkBox_pstate0_2</tabstop>
  <tabstop>checkBox_pstate3_2</tabstop>
  <tabstop>checkBox_pstate1_2</tabstop>
  <tabstop>checkBox_pstate2_2</tabstop>
  <tabstop>verticalSlider_pstate4_gpu_offset_2</tabstop>
  <tabstop>verticalSlider_pstate0_gpu_offset_2</tabstop>
  <tabstop>verticalSlider_pstate2_gpu_offset_2</tabstop>
  <tabstop>checkBox_pstate4_2</tabstop>
  <tabstop>verticalSlider_pstate3_gpu_offset_2</tabstop>
  <tabstop>verticalSlider_pstate1_gpu_offset_2</tabstop>
  <tabstop>verticalSlider_pstate5_gpu_offset_2</tabstop>
  <tabstop>checkBox_pstate5_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
